docs/translations/translations-lazy-loading.md

```diff
@@ -0,0 +1,150 @@
+# Translations for lazy-loaded components
```

Comment:
Should we update this file name and its reference in `docs/translations/README.md` to match with the new title?